app/models/project.rb

```diff
@@ -1,4 +1,23 @@
 class Project < ApplicationRecord
   validates_presence_of :name, :material
   belongs_to :challenge
+  has_many :contestant_projects
+  has_many :contestants, through: :contestant_projects
+
+  def project_challenge_theme
+    challenge.theme
```

Comment:
Totally fine to encapsulate this BUT it's very okay to do `project.challenge.theme` in a view, for the future